website/src/app/pages/data-distributions/data-distributions.component.html

```diff
@@ -17,4 +17,17 @@
     [options]="{}"
     [autosize]="{ width: true, height: false }"
   ></ngx-vega>
+</div> -->
+
+<div class="data-block">
+  <ngx-vega
```

Comment:
Add one spec per type for demonstration purposes here. Daniel will replace with his structuring later.